scripts/semantic-release/assets/utils/intent.ts

```diff
@@ -298,13 +304,13 @@ export function hashReward(reward: RewardType): Hex {
  * });
  * console.log(`Intent hash: ${hashes.intentHash}`);
  */
-export function hashIntent(intent: IntentType): {
+export function hashIntent(route: RouteType, reward: RewardType): {
```

Comment:
You'd need the destination chain, too. `intentHash = keccak256(destinationChain, routeHash, rewardHash)`.